src/wizard.ts

```diff
@@ -64,22 +50,7 @@ export abstract class WizardBase {
                 this.beforeExecute(step, i);
                 await this.steps[i].execute();
             } catch (err) {
-                this.sendErrorTelemetry(step, err);
-                this.onExecuteError(err, step, i);
-                if (err instanceof UserCancelledError) {
-                    this._result = {
-                        status: 'Cancelled',
-                        step: step,
-                        error: err
-                    };
-                } else {
-                    this._result = {
-                        status: 'Faulted',
-                        step: step,
-                        error: err
-                    };
-                }
-                return this._result;
+                this.onExecuteError(err, step);
```

Comment:
Can we remove onExecuteError and just add this code? It's the same in every place:
```
if (error instanceof UserCancelledError) {
    return;
}
throw new WizardFailedError(error, step.stepTitle, step.stepIndex);
```

Also - onExecuteError shouldn't display anything because initAsyncCommand does that

Reply:
It displays on the output pane so I didn't want to remove it without asking.  I see some value in having it on the output in case somebody wanted to investigate into the error message more and needed a reference for later.

Reply:
Okay makes sense - but that should apply for every step, though, correct? So we could still get rid of the 'onExecuteError' thing and display in output pane here?